index.html

```diff
@@ -286,7 +286,7 @@ <h2 class="text-2xl font-medium tracking-normal text-[#F3702F]">Flawless perform
                 Size</dt>
               <dd
                 class="order-first text-3xl font-semibold tracking-tight text-white sm:text-5xl"
-                data-target="40">0</dd>
+                data-target="+40">+40</dd>
```

Comment:
This doesn't work (at least on my computer)
I added a div for the "+" and "%" and it's working on my end

```
<dd class="order-first text-3xl font-semibold tracking-tight text-white sm:text-5xl flex justify-center">
                  <div>+</div>
                  <div data-target="40">0</div>
                </dd>
```